CHANGELOG.md

```diff
@@ -31,3 +31,5 @@ All notable, unreleased changes to this project will be documented in this file.
 - Fixed order-level discounts handling when using tax app for tax calculation - #16696 by @zedzior
 - Fixed bug when manual line discount doesn't override line-level vouchers - #16738 by @zedzior
 - Skipped obsolete payload save and cleanup for successful sync webhooks - #16632 by @cmiacz
+- Removed support for the django-debug-toolbar debugging tool and the `ENABLE_DEBUG_TOOLBAR` env variable - #16902 by @patrys
+- Fixed playground not displaying docs if api is hidden behind reverse proxy - #16810 by @jqob
```

Comment:
I think something happened with this CHANGELOG, you should be adding just your version here 🤔

Reply:
I am not sure, since that's the only changelog I can see on my branch.